CMake/cdat_modules/cycler_pkg.cmake

```diff
@@ -0,0 +1,10 @@
+set(CYCLER_MAJOR 0)
+set(CYCLER_MINOR 9)
+set(CYCLER_PATCH 0)
+set(CYCLER_VERSION ${CYCLER_MAJOR}.${CYCLER_MINOR}.${CYCLER_PATCH})
+set(CYCLER_URL ${LLNL_URL})
+set(CYCLER_GZ cycler-${CYCLER_VERSION}.tar.gz)
+set(CYCLER_MD5 c10ade5ca3f0aadf575eb25203b225a5)
+set(CYCLER_SOURCE ${CYCLER_URL}/${CYCLER_GZ})
+
+add_cdat_package_dependent(Cycler "" "" OFF "CDAT_BUILD_LEAN" OFF)
```

Comment:
@doutriaux1 not certain that this magic is correct - I used the template from `Cython`